homeassistant/components/lock/wink.py

```diff
@@ -206,4 +205,4 @@ def dict_value_to_key(dict_map, comp_value):
     for key, value in dict_map.items():
         if value == comp_value:
             return key
-    return STATE_UNKNOWN
+    return None
```

Comment:
The changed function here creates state attributes, so this will be a breaking change. State attributes are not parsed, the way state is parsed, converting `None` to `STATE_UNKNOWN` in the base entity class.